llpc/translator/lib/SPIRV/libSPIRV/SPIRVModule.cpp

```diff
@@ -490,21 +490,23 @@ void SPIRVModuleImpl::postProcessExecutionModeId() {
   for (auto ExecModeId : ExecModeIdVec) {
     SPIRVExecutionModeId *E = static_cast<SPIRVExecutionModeId *>(ExecModeId);
     auto M = E->getExecutionMode();
+    auto tid = E->getTargetId();
+    auto ops = E->getOperands();
+    SPIRVExecutionMode *execMode = nullptr;
     switch (M) {
     case ExecutionModeLocalSizeId: {
```

Comment:
It's a bit unusual to have a `switch` statement with only one case. Wouldn't `if` work better here?

Reply:
it means to reserve cases for the spirv ExecutionMode for the internal project.